.github/workflows/test_qlib_from_source_fast.yml

```diff
@@ -8,42 +8,57 @@ on:
 
 jobs:
   build:
-    timeout-minutes: 120
+
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [windows-latest, ubuntu-18.04, ubuntu-20.04]
+        os: [windows-latest, ubuntu-18.04, ubuntu-20.04, macos-11, macos-latest]
         # not supporting 3.6 due to annotations is not supported https://stackoverflow.com/a/52890129
         python-version: [3.7, 3.8]
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Test qlib from source fast
+      uses: actions/checkout@v2
 
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Lint with Black
+    - name: Install pytorch according to the system
```

Comment:
`Install PyTorch according to the system `  Appears several times.
It would be more meaningful if we can use different names.